packages/cli/src/commands/setup/auth/__tests__/__templates__/AppWithExplicitReturn.template

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from 'react'
+import { FatalErrorBoundary, RedwoodProvider } from '@redwoodjs/web'
+import { RedwoodApolloProvider } from '@redwoodjs/web/apollo'
+
+import FatalErrorPage from 'src/pages/FatalErrorPage'
+import Routes from 'src/Routes'
+
+import './index.css'
+
+const App = (props) => {
+  const { cache } = props
+
+  useEffect(() => {
+    console.log('Running my custom useEffect hook once.')
```

Comment:
```suggestion
    console.log('Running my custom useEffect hook on each render.')
```

I know it doesn't really matter, just bugged me that it was wrong :)

Reply:
hahah of course!